starter-code/src/main.js

```diff
@@ -42,9 +43,57 @@ $(document).ready(function(){
   // Add all the div's to the HTML
   document.getElementById('memory_board').innerHTML = html;
   // Bind the click event of each element to a function
+
+var countClicker = 0;
+var $pairsClicked = $('#pairs_clicked');
+var $pairsGuessed = $('#pairs_guessed');
+
 $('.back').on('click', function () {
-   
+  $self = $(this);
+  $self.addClass('active');
+  countClicker++;
+  $self.css('display','none');
+  $self.next().css('display', 'inline');
+  if (countClicker == 1) {
+    cardOneName = $self.attr('name');
+  } else {
+    cardTwoName = $self.attr('name');
+  }
+  if (countClicker == 2) {
+    var pairChecker = memoryGame.checkIfPair(cardOneName, cardTwoName);
+    if (pairChecker == true) {
+      $activeCards = $('.active');
+      $activeCards.removeClass('active');
+      var decideToFinish = memoryGame.finished();
+      if (decideToFinish) {
+        alert('You won!');
+        reset();
+        
+      }
+    } 
+    else {
+      turnWrongPair();
+    }
+    $pairsClicked.text(memoryGame.pairsClicked);
+    $pairsGuessed.text(memoryGame.pairsGuessed);
+    countClicker = 0;
+  }
```

Comment:
```
+  if (countClicker == 2) {
 +    var pairChecker = memoryGame.checkIfPair(cardOneName, cardTwoName);
 +    if (pairChecker == true) {
 +      $activeCards = $('.active');
 +      $activeCards.removeClass('active');
 +      var decideToFinish = memoryGame.finished();
 +      if (decideToFinish) {
 +        alert('You won!');
 +        reset();
 +        
 +      }
 +    } 
 +    else {
 +      turnWrongPair();
 +    }
 +    $pairsClicked.text(memoryGame.pairsClicked);
 +    $pairsGuessed.text(memoryGame.pairsGuessed);
 +    countClicker = 0;
 +  }
```
All of this can be inside the first else.